docs/whats-new.md

```diff
@@ -24,6 +24,18 @@ Highlights:
 - Internal: Tests updated to use `vitest` instead of `jest`
 - Website: Search restored.
 
+### `@deck.gl-community/editable-layers`
+
+- DrawPolygonMode: Added `allowHoles` configuration to enable drawing polygon holes within existing polygons
+- DrawPolygonMode: Enhanced hole creation with validation to prevent overlapping or nested holes
+- DrawPolygonMode: Added comprehensive edit types for hole operations (`addHole`, `invalidHole`)
+
+Breaking Changes:
+- DrawPolygonMode: `preventOverlappingLines` configuration renamed to `allowSelfIntersection` for consistency
+  - **Migration**: Update your modeConfig from `{preventOverlappingLines: false}` to `{allowSelfIntersection: true}`
```

Comment:
- We typically put the migration details in upgrade-guide.md.
- Generally we want whats-new.md to be an exciting read, getting users excited about the new improvements, rather than a eye-glazing list of technical details :-D